documentation/modules/operators/proc-changing-topic-operator-mode.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-changing-topic-operator-mode-{context}']
+= (Preview) Switching between Topic Operator modes
+
+[role="_abstract"]
+You can switch between topic management modes when upgrading or downgrading Strimzi, or when using the same version of Strimzi, as long as the mode is supported for that version. 
+
+.Switching from bidirectional to unidirectional topic management mode
+
+. xref:ref-operator-unidirectional-topic-operator-feature-gate-str[Enable the `UnidirectionalTopicOperator` feature gate]. 
++
+The Cluster Operator deploys the Entity Operator with the Topic Operator in unidirectional topic management mode.
+
+. The internal topics that support the Topic Operator running in bidirectional topic management mode are no longer required, so you can delete the `KafkaTopic` resources to manage them:
++
+[source,shell,subs=+quotes]
+----
+kubectl delete $(kubectl get kt -o name | grep strimzi-store-topic) && kubectl delete $(kubectl get kt -o name | grep strimzi-topic-operator)
```

Comment:
Do we want to add the namespace flag to avoid accidentally breaking another topic operator running in a separate namespace that happened to be in the kubectl context?